README.md

```diff
@@ -1,6 +1,25 @@
-# oncotree
+# OncoTree
+
+## General
+
+The OncoTree was developed at [Memorial Sloan Kettering Cancer Center](https://www.mskcc.org/) (MSK) as a resource for standardizing detailed cancer type information and mapping these details to unique accessions called OncoTree codes.
+
+OncoTree codes are linked to every study imported into the [cBioPortal for Cancer Genomics](https://www.cbioportal.org/), where each sample is annotated with its own OncoTree code corresponding to its respective cancer type.
+
+These codes are also used by [OncoKB](http://oncokb.org/), a precision oncology knowledge base developed at MSK containing information about the effects and treatment implications of specific cancer gene alterations.
+
+
+Read about our latest developments on our [News page](/docs/News.md).
+
+## OncoTree Version Converter Tool
+
+The OncoTree Version Converter is a tool developed to facilitate forwards- and backwards-mapping between OncoTree release versions. To learn more about the OncoTree Converter tool, please refer to the [OncoTree Converter Wiki page](/docs/OncoTree-Version-Converter.md).
```

Comment:
I would mention what is being converted (oncotree codes).

Reply:
I agree, but didn't notice that myself!